app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LaunchScreenActivity">
+
+    <TextView
+        android:id="@+id/txt_sign"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_5x"
+        android:layout_marginBottom="@dimen/margin_3x"
+        android:fontFamily="@font/poppins_regular"
+        android:text="@string/launch_screen_txt_sign_in"
+        android:textColor="@color/s_60"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/txt_welcome"
+        app:layout_constraintEnd_toEndOf="@+id/button"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/txt_welcome"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_3x"
+        android:layout_marginEnd="@dimen/margin_3x"
+        android:layout_marginTop="@dimen/margin_10x"
+        android:fontFamily="@font/comfortaa_regular"
+        android:text="@string/launch_screen_txt_welcome"
+        android:textAlignment="center"
+        android:textColor="@color/s_60"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:lineHeight="20sp" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/margin_3x"
+        android:layout_marginTop="@dimen/margin_3x"
+        android:layout_marginRight="@dimen/margin_3x"
+        android:contentDescription="@string/launch_screen_image_launch"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/txt_welcome"
+        app:srcCompat="@drawable/launcher_image" />
+
+    <TextView
+        android:id="@+id/txt_message"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_10x"
+        android:layout_marginEnd="@dimen/margin_10x"
+        android:layout_marginTop="@dimen/margin_2.5x"
+        android:fontFamily="@font/comfortaa_regular"
+        android:text="@string/launch_screen_txt_subtext"
+        android:textAlignment="center"
+        android:textColor="@color/s_60"
+        android:textSize="16sp"
```

Comment:
Don't we have a style for TextView in styles.xml? Please also fix other TextViews in this xml file